content/docs/user-guide/components/reference/atom/bloom.md

```diff
@@ -1,20 +1,39 @@
 ---
+linkTitle: Bloom
 title: Bloom Component
-linktitle: Bloom
-description: "Open 3D Engine (O3DE) Bloom component reference."
+description: The Bloom component, which simulates real-world light bleeding (glow), is provided by the Atom Gem in Open 3D Engine (O3DE). 
 toc: true
 ---
 
-The **Bloom component** simulates real-world light bleeding, or glow.
+The **Bloom** component creates *bloom*, a post-processing effect that simulates real-world light bleeding, or glow.
 
 
-## Provider ##
+## Provider
 
-[Atom Gem](/docs/user-guide/gems/reference/rendering/atom/atom/)
+[Atom Gem](/docs/user-guide/gems/reference/rendering/atom/atom)
+
+
+## Dependencies
+
+[PostFX Layer component](./postfx-layer)
 
 
 ## Properties
 
-{{< todo issue=https://github.com/o3de/o3de.org/issues/717 >}}
-Add properties.
-{{< /todo >}}
\ No newline at end of file
+![Bloom component interface](/images/user-guide/components/reference/atom/bloom-component-ui.png)
+
+
+### Bloom properties
+
+| Property | Description | Value | Default |
+| - | - | - | - |
+| **Enable Bloom** | If enabled, activates the bloom effect. | Boolean | `Enabled` |
+| **Overrides** | If enabled, all Bloom component properties will be set to the values specified in the **Overrides** property group. | Boolean | `Enabled` |
```

Comment:
I don't get it. Why are overrides on by default? Seems like a strange UI/UX choice to me.

Reply:
Also, it doesn't seem to be a simple value override to me. Looking at Intensity, for example: The normal range is 0 to 10000. (Anything above 2 produces absurd-looking results, but whatever.) If I set Intensity to 2.5 and the Intensity Override factor to 0.0, you'd think that I'd see a difference when I toggle Enable Override, but I don't. Also, the scale of the Intensity Override is only 0 to 1. If it were a simple value replacement override, the scale should be the same.

Reply:
https://github.com/o3de/o3de/issues/5993

Reply:
An open issue related to the intensity slider: https://github.com/o3de/o3de/issues/5993.